covsirphy/cleaning/jhu_complement.py

```diff
@@ -145,6 +145,9 @@ def _pre_processing(self, subset_df):
                 Index: Date (pandas.TimeStamp)
                 Columns: Confirmed, Fatal, Recovered
         """
+        sel_valid_R = subset_df[self.C] - subset_df[self.F] < subset_df[self.R]
```

Comment:
Just to confirm, do you mean "invalid" by `sel_valid_R`?

Reply:
Yes invalid